kafka-ui-react-app/src/serviceWorker.ts

```diff
@@ -17,7 +17,7 @@ const isLocalhost = Boolean(
     window.location.hostname === '[::1]' ||
     // 127.0.0.0/8 are considered localhost for IPv4.
     window.location.hostname.match(
-      /^127(?:\.(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3}$/
+      /^127(?:\.(?:25[0-5]|2[0-4]\d|[01]?\d\d?)){3}$/
```

Comment:
I'm not sure we should touch this file. Pls skip it

Reply:
sonar suggested to change it ilke this,
ok will bring this changes back

Reply:
we can mark/skip some sonar suggestions btw

Reply:
I've added it to the excluded files list